ajax/libs/blissfuljs/package.json

```diff
@@ -17,19 +17,16 @@
   ],
   "author": "Lea Verou",
   "license": "MIT",
-  "homepage": "http://blissfuljs.com",
-  "autoupdate": {
-    "source": "git",
-    "target": "git://github.com/LeaVerou/bliss.git",
-    "fileMap": [
-      {
-        "basePath": "",
-        "files": [
-          "bliss.js",
-          "bliss.min.js",
-          "bliss.shy*.js"
-        ]
-      }
-    ]
-  }
+  "homepage": "https://blissfuljs.com",
```

Comment:
Let's do this change using another commit because it's meta-data.
Please notice this commit should be done before new version add & auto-update.
You may use `reset --soft HEAD^` to reserve your changes in unstaged area and them redo commits.
Here is a suggestion for first commit:
```
Add blissfuljs@1.0.5 w/ npm auto-update
close #12318
```

second commit
```
Update blissfuljs homepage to use https, cc #12318
```

Refer https://github.com/cdnjs/cdnjs/pull/11551#pullrequestreview-59526021